app/src/main/java/ch/epfl/sdp/CoronaGame.java

```diff
@@ -35,7 +35,7 @@
     public static boolean IS_ONLINE = TRUE;
     public static boolean IS_NETWORK_DEBUG = FALSE;
     private static Context context;
-    private static int DEMO_SPEEDUP = 1;
+    private static int DEMO_SPEEDUP = 10;
```

Comment:
Why put the demo speedup on the master ?
I think it suffices that if someone wants to make a demo he changes it locally.

Reply:
I'd like speedup the released APK and that's a way to do it without changing all the constants